recipe/meta.yaml

```diff
@@ -1,44 +1,64 @@
-{% set version = "2018_20170919" %}
+{% set vmajor = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MAJOR +(\d+)").group(1)|int %}
+{% set vminor = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MINOR +(\d+)").group(1)|int %}
+{% set vinterface = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_INTERFACE_VERSION +(\d+)").group(1)|int %}
+{% set version = "%d.%d.%d"%(vmajor,vminor,vinterface%1000) %}
```

Comment:
I think these are confusing the linter and seems to confuse `conda-smithy` during re-rendering. Is there some way to guard against them and possibly provide a default value for the version?

Reply:
well, we can go back to plain old good numbers and change them along with the git_tag with each release - not a big deal. I'll update it along with removing the Python part